packages/wrangler/src/api/upgrade.ts

```diff
@@ -0,0 +1,94 @@
+import { writeFileSync, existsSync } from "fs";
+import path from "path";
+import { execaSync } from "execa";
+import { fetch } from "undici";
+import { findWranglerToml } from "../config";
+import { confirm } from "../dialogs";
+import { printWranglerBanner } from "../index";
+import { logger } from "../logger";
+import { readFileSync } from "../parse";
+
+import type { PackageJSON } from "../parse";
+
+export async function upgradeWrangler({
+	yesFlag = false,
+}: {
+	yesFlag: boolean;
+}) {
+	await printWranglerBanner();
+	let packageJsonData: undefined | PackageJSON;
+
+	const pathToWorker = findWranglerToml();
```

Comment:
```suggestion
	const pathToWorkerConfig = findWranglerToml();
```

Reply:
I am using it to check where the Worker is, I don't actually care about the config.

Reply:
Absolutely, but this variable contains the path to the worker config, right?